Contxtful RTD Provider - Updated Documentation (#5322)
prebid/prebid.github.io: dev-docs/modules/contxtfulRtdProvider.md

          "customer": "Contact contact@contxtful.com for the customer ID",
          "hostname": "api.receptivity.io", // Optional, default: "api.receptivity.io"
          "bidders": ["bidderCode1", "bidderCode", "..."], // list of bidders
          "adServerTargeting": true, // Optional, default: true
        }
      }
    ]
  }
});
```

## Parameters

{: .table .table-bordered .table-striped }
| Name                | Type     | Scope    | Description                                |
|---------------------|----------|----------|--------------------------------------------|
| `version`           | `String` | Required | Specifies the version of the Contxtful     |
|                     |          |          | Receptivity API.                           |
| `customer`          | `String` | Required | Your unique customer identifier.           |
| `hostname`          | `String` | Optional | Target URL for CONTXTFUL external JavaScript file. Default is "api.receptivity.io". Changing default behaviour is not recommended. Please reach out to [contact@contxtful.com](mailto:contact@contxtful.com) if you experience issues. |

> | `adServerTargeting` | `Boolean`| Optional | Enables the `getTargetingData` to inject targeting value in ad units. Setting this parameter to true enables the feature. [Default] Setting this parameter to false disables the feature.       |

Reviewer: ```suggestion
| `adServerTargeting` | `Boolean`| Optional | Enables the `getTargetingData` to inject targeting value in ad units. Setting to true enables the feature, false disables the feature. Default is true      |
```

Reviewer 2: Done. see https://github.com/prebid/prebid.github.io/pull/5322/commits/98bfa43746b94d3e40a619df087910a0e4bb4c91

## Usage: Injection in Ad Servers

The `contxtfulRtdProvider` module loads an external JavaScript file and authenticates with Contxtful APIs. The `getTargetingData` function then adds a `ReceptivityState` to each ad slot, which can have one of two values: `Receptive` or `NonReceptive`.

```json
{
  "adUnitCode1": { "ReceptivityState": "Receptive" },
  "adUnitCode2": { "ReceptivityState": "Receptive" }
}
```

This module also integrates seamlessly with Google Ad Manager, ensuring that the `ReceptivityState` is available as early as possible in the ad serving process.

## Usage: Injection in ortb2 for bidders

Setting the `bidders` field in the configuration parameters enables Receptivity in the `ortb2` object through `getBidRequestData` for all the listed bidders.
On a Bid Request Event, all bidders in the configuration will inherit the Receptivity data through `ortb2`
Default is `[]` (an empty array)

> RECOMMENDED : Add all the bidders active like this `["bidderCode1", "bidderCode", "..."]`

Reviewer: Would need a `return` before this line to create a distinct paragraph

Reviewer 2: Done. see https://github.com/prebid/prebid.github.io/pull/5322/commits/98bfa43746b94d3e40a619df087910a0e4bb4c91
